### ISSUE TITLE
TypeError: Missing 'capacity_constraints_max' argument 

### ISSUE THREAD

**Issue by t0yv0** [pulumi/pulumi-ec#172]
### What happened?

Routine upgrade of the bridge and code-generators https://github.com/pulumi/pulumi-ec/pull/171 yields an error in one of the regression tests.

```
    File "/tmp/p-it-fv-az589-8-py-ab5a582d-661333913/venv/lib/python3.9/site-packages/pulumi/runtime/rpc.py", line 989, in <listcomp>
      translate_output_properties(
    File "/tmp/p-it-fv-az589-8-py-ab5a582d-661333913/venv/lib/python3.9/site-packages/pulumi/runtime/rpc.py", line 949, in translate_output_properties
      return _types.output_type_from_dict(typ, translated_values)
    File "/tmp/p-it-fv-az589-8-py-ab5a582d-661333913/venv/lib/python3.9/site-packages/pulumi/_types.py", line 581, in output_type_from_dict
      return cls(**args)  # type: ignore
    File "/home/runner/work/pulumi-ec/pulumi-ec/sdk/python/bin/pulumi_ec/outputs.py", line 7407, in __init__
      GetStackEnterpriseSearchResult._configure(
    File "/home/runner/work/pulumi-ec/pulumi-ec/sdk/python/bin/pulumi_ec/outputs.py", line 7428, in _configure
      raise TypeError("Missing 'capacity_constraints_max' argument")
  TypeError: Missing 'capacity_constraints_max' argument
  ```

### Example

N/A

### Output of `pulumi about`

N/A

### Additional context

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
@t0yv0 Are you looking into this or should I assign @iwahbe (ops)?

**Comment by justinvp** [comment]
I haven't drilled into the details here, but this will be addressed with an upgrade to v3.91.1 of Pulumi codegen, which reverts the change that introduced `_configure`.

**Comment by iwahbe** [comment]
Should be fixed by https://github.com/pulumi/pulumi-ec/pull/178.
